common/common-protobuf/src/test/java/io/protostuff/runtime/model/ModelProtobuf.java

```diff
@@ -3,6 +3,8 @@
 
 package io.protostuff.runtime.model;
```

Comment:
this file is model generated by protobuf, just for compatible test
it's better not change it.